brainscore_vision/nwb_integration/create_assembly.py

```diff
@@ -0,0 +1,203 @@
+import re, json
+import numpy as np
+import pandas as pd
+import xarray as xr
+
+from brainio.assemblies import NeuronRecordingAssembly
+from brainio.stimuli import StimulusSet
+from brainscore_vision import load_ceiling
+from brainscore_vision.metric_helpers.transformations import CrossValidation
+from pynwb.file import NWBFile
+from typing import Tuple
+
+
+def get_neuroids(nwb_file: NWBFile, subject: str) -> pd.DataFrame:
+    #-----------------------------------------------------------------------------------------------------------------------------
+    # Get electrode metadata from nwb file.
+    #-----------------------------------------------------------------------------------------------------------------------------
+    data_list = []  
+    for i in range(len(nwb_file.electrodes['location'])):
+        data_dict       = {}
+        location_item   = nwb_file.electrodes['location'][i]
+        group_item      = nwb_file.electrodes['group'][i] 
+        bank_item       = nwb_file.electrodes['group_name'][i] 
+        label_item      = nwb_file.electrodes['label'][i]
+        try: label_item = int(nwb_file.electrodes['label'][i].split('_')[0])
+        except: label_item = nwb_file.electrodes['label'][i]
+
+        location_match = re.search(r'\[(\d+).0, (\d+).0, (\d+).0\]', location_item)
+        if location_match:
+            data_dict['col'] = location_match.group(2)
```

Comment:
If `re` does not find a match, I don't think the `data_dict[row]`, `data_dict[col]`, and `data_dict[elec]` will be set - might be worth adding an `else` here that sets those values to `None`